sdk/python/kfp/compiler/main.py

```diff
@@ -47,12 +57,19 @@ def parse_arguments():
   parser.add_argument('--disable-type-check',
                       action='store_true',
                       help='disable the type check, default is enabled.')
+  parser.add_argument('--enable-telemetry',
+                      type=_str2bool,
+                      nargs='?',
+                      const=True,
+                      default=True,
+                      help='allow adding pod labels, default is enabled.')
```

Comment:
You could simplify this with the following [similarly to `--disable-type-check` argument]
```suggestion
  parser.add_argument('--disable-telemetry',
                      action='store_true',
                      help='disable adding pod labels, default is enabled.')
```
(and then pass `not args.disable_telemetry` in `compile_pyfile` and `compile_package`)

Reply:
Thanks! Done.

Reply:
You're welcome! You should now be able to remove the `_str2bool()` function, too.